python/jupytercad_core/pyproject.toml

```diff
@@ -42,7 +42,7 @@ fields = ["description", "authors", "urls"]
 
 [tool.hatch.build.targets.sdist]
 artifacts = ["jupytercad_core/labextension"]
-exclude = [".github", "binder"]
+exclude = [".github", "binder", "node_modules"]
```

Comment:
Question: is `binder` here related to the thing - launch the binder that was being used on PRs before?
Are we still using that?

Reply:
we are not using it in the PR but it can be useful for others who want to deploy jupytercad to mybinder